gtk/transmission-gtk.appdata.xml.in

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2014 Richard Hughes <richard@hughsie.com>
+Copyright 2017 Endless Mobile, Inc.
```

Comment:
It was. I worked on this as part of making a [Flatpak of Transmission](https://github.com/endlessm/transmission-flatpak) to include it in Endless OS. How would you feel about adding the Flatpak build script to this repo, too? It wouldn't commit Transmission upstream to publishing a Flatpak, of course.